src/main/groovy/io/openliberty/tools/gradle/Liberty.groovy

```diff
@@ -164,8 +165,21 @@ class Liberty implements Plugin<Project> {
            } else {
                return new File(project.liberty.baseDir, 'wlp')
            }
-        } else {
-           return new File(project.liberty.installDir)
+        } else { // installDir is specified
+            String installDirPath = checkAndAppendWlp(project)
+            File installDir = new File(installDirPath)
+
+            return installDir;
+        }
+    }
+
+    private static String checkAndAppendWlp(Project project) {
+        String installDir = project.liberty.installDir
+        if (installDir.endsWith("wlp")) {
+            return installDir
+        } else { // not valid wlp dir
+            project.getLogger().warn(MessageFormat.format("The installDir {0} path does not reference a wlp folder. Checking for a valid Liberty installation at path {0}/wlp instead.", installDir))
```

Comment:
Let's change this warning message since we are no longer doing the validation checking for the installation here.

Reply:
How about just removing the word "valid"?

Reply:
It's not checking the installation here. Change the second sentence to "Using path {0}/wlp instead."